app/models/course/lesson_plan_item.rb

```diff
@@ -0,0 +1,21 @@
+class Course::LessonPlanItem < ActiveRecord::Base
+  actable
+  stampable
+  validates :base_exp, numericality: { only_integer: true }
+  validates :time_bonus_exp, numericality: { only_integer: true }
+  validates :extra_bonus_exp, numericality: { only_integer: true }
+
+  after_initialize :set_default_values
```

Comment:
<code>after_initialize :set_default_values if: :new_record?</code> so this function only been called upon creation.